pkg/controller/release/release_controller.go

```diff
@@ -488,6 +514,8 @@ func (c *Controller) getAssociatedApplicationName(rel *shipper.Release) (string,
 	return appref.Name, nil
 }
 
+// getAssociatedApplicationKey returns an application key in the format:
+// <namespace> / <application name>
```

Comment:
I would take the spaces between namespace and application name in there literally. Changing to `"<namespace>/<application name>"` would be more explicit regarding the key's format.